compiler/util/files.cpp

```diff
@@ -69,17 +69,36 @@ static const char* intDirName        = NULL;
 
 static const int   MAX_CHARS_PER_PID = 32;
 
-void addLibInfo(const char* libName) {
+//
+// Convert a libString of the form "-Lfoo:bar:baz" to entries in libFlag[]
+//
+void addLibInfo(const char* libStringOrig) {
   static int libSpace = 0;
+  const char* flag = "";    // do we need a '-L' flag?  (initially, no)
+  char* colon;              // used to refer to ':'s in libString
+  int libStringLen = strlen(libStringOrig)+1;
```

Comment:
Is there a strong reason for not just using `strdup()` here?  It doesn't look like libStringLen is needed other than to drive the `malloc()` and `strncpy()`.

Reply:
Just my own ignorance, thanks for the suggestion!